src/cfml/system/services/ServerService.cfc

```diff
@@ -128,6 +128,7 @@ component accessors="true" singleton {
 			'trayOptions' : duplicate( d.trayOptions ?: [] ),
 			'trayEnable' : d.trayEnable ?: true,
 			'dockEnable' : d.dockEnable ?: true,
+			'preferedBrowser' : d.preferedBrowser ?: '',
```

Comment:
I don't think we need to set this at a server level.  It should just be a CommandBox-wide setting that is passed along to runwar.